web/satellite/src/components/account/billing/paymentMethods/TokenDepositSelection2.vue

```diff
@@ -0,0 +1,326 @@
+import {PaymentsHistoryItemType} from "@/types/payments";
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+<template>
+    <div class="form-container">
+        <div v-if="!isCustomAmount" class="selected-container">
+            <div class="selected-container__label-container" @click="open">
+                <p class="selected-container__label-container__label">{{ current.label }}</p>
+                <div :class="`selected-container__label-container__svg ${isSelectionShown ?'down': 'up'}`">
+                    <svg width="14" height="8" viewBox="0 0 14 8" fill="none" xmlns="http://www.w3.org/2000/svg">
+                        <path fill-rule="evenodd" clip-rule="evenodd" d="M0.372773 0.338888C0.869804 -0.112963 1.67565 -0.112963 2.17268 0.338888L7 4.72741L11.8273 0.338888C12.3243 -0.112963 13.1302 -0.112963 13.6272 0.338888C14.1243 0.790739 14.1243 1.52333 13.6272 1.97519L7 8L0.372773 1.97519C-0.124258 1.52333 -0.124258 0.790739 0.372773 0.338888Z" fill="#000000" />
+                    </svg>
+                </div>
+            </div>
+        </div>
+        <label v-if="isCustomAmount" class="label">
+            <input
+                v-model="customAmount"
+                v-number
+                class="custom-input"
+                placeholder="Enter Amount in USD"
+                @input="onCustomAmountChange"
+            >
+            <p v-if="customAmount" class="label__sign">$</p>
+            <div class="input-svg" @click.stop="closeCustomAmountSelection">
+                <svg width="14" height="8" viewBox="0 0 14 8" fill="none" xmlns="http://www.w3.org/2000/svg">
+                    <path fill-rule="evenodd" clip-rule="evenodd" d="M0.372773 0.338888C0.869804 -0.112963 1.67565 -0.112963 2.17268 0.338888L7 4.72741L11.8273 0.338888C12.3243 -0.112963 13.1302 -0.112963 13.6272 0.338888C14.1243 0.790739 14.1243 1.52333 13.6272 1.97519L7 8L0.372773 1.97519C-0.124258 1.52333 -0.124258 0.790739 0.372773 0.338888Z" fill="#000000" />
+                </svg>
+            </div>
+        </label>
+        <div
+            v-if="isSelectionShown"
+            v-click-outside="close"
+            class="options-container"
+        >
+            <div
+                v-for="option in paymentOptions"
+                :key="option.label"
+                class="options-container__item"
+                @click.prevent.stop="select(option)"
+            >
+                <div v-if="isOptionSelected(option)" class="options-container__item__svg">
+                    <svg width="15" height="13" viewBox="0 0 15 13" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
same

Reply:
This is following the original design pattern of the component this was based on.